### PR TITLE
Remove convergence and interactor from monorepo

### PR DESCRIPTION
**Pull request by minkimcello**
## Motivation
Just following orders. 👮 

Interactor and convergence packages are outdated and no longer being used so we're removing them for tidying up purposes.

## Approach
Removed `interactor` and `convergence` from the packages directory and updated `test` + `release` workflows.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       with:
         before_all: yarn prepack
         npm_publish: yarn publish
-        ignore: packages/website packages/convergence packages/interactor packages/cli
+        ignore: packages/website packages/cli
```
Comment (cowboyd) [inline .github/workflows/release.yml:19]: Why are we ignoring `cli`?

Comment (minkimcello) [reply .github/workflows/release.yml:19]: Hmmm I'm not sure. I was wondering about that too. I checked to see who added it and turns out I added it from the get go ([PR 132](https://github.com/thefrontside/bigtest/pull/132/)). 🤷 

I'll go ahead and remove it.

**Comment by taras** [comment]
@cowboyd should we add a mechanism to release action to somehow deprecate packages?

**Comment by cowboyd** [comment]
> @cowboyd should we add a mechanism to release action to somehow deprecate packages?

That would actually be awesome.... how do you deprecate an NPM package?

**Comment by taras** [comment]
@cowboyd https://docs.npmjs.com/cli/deprecate

**Comment by cowboyd** [comment]
Right. In that case, what about adding a `deprecated` flag to the package.json and then we have our action synchronize the deprecation state with.

**Comment by minkimcello** [comment]
Moved discussion to [this](https://github.com/thefrontside/actions/issues/57) issue.

**Pull request merged**